packages/api/src/hooks/useSortedMT5Accounts.ts

```diff
@@ -22,6 +22,7 @@ const useSortedMT5Accounts = () => {
                     ...created_account,
                     /** Determine if the account is added or not */
                     is_added: true,
+                    is_max_account_created: all_available_mt5_accounts?.length === mt5_accounts?.length,
```

Comment:
```suggestion
```
is this needed? u already do a max length check below

Reply:
nope